docs/tools.md

```diff
@@ -0,0 +1,190 @@
+# The Lean tool chain
+
+## Introduction
+
+In this document we explain how all the various pieces of the Lean tools ecosystem fit together. 
+Understanding this precisely is not required, but it will help if you decide to explore non-recommended setups.
+In particular we will gradually introduce new pieces in the story, starting with a bare bone Lean. 
+This is *not* the recommended setup.
+Also this document is long, that's the price for not wanting to blindly follow the recommended setup instructions.
+
+## Installing Lean and checking files
+
+### Installing Lean
+
+Of course the main piece is Lean itself. You can download Lean binaries for your OS [here](https://github.com/leanprover-community/lean/releases). 
+This contains folders `bin`, `include` and `lib`.
+The `bin` folder contains a `lean` executable which you can run.
+
+That's all, Lean is installed!
+
+You can make it slightly more convenient to use by ensuring this `bin` folder is in your path, or maybe by creating a link to 
```

Comment:
Maybe, we could mention popular distros installation? Archlinux, Gentoo, NixOS, etc. I don't know if it makes sense.
I mean, as we're in a non-recommended setup, we could go as well explain how to have your own patched version of Lean and compile it.

Reply:
I don't think the linux Lean binaries assume anything about your distro. It's true that we see people on Zulip from time to time that insist on compiling Lean, but this is somewhat orthogonal to what is discussed in this document. This section is partly an answer to people who don't understand anything about how tools fit together and complain on Zulip that Lean is hard to install (or depend on python or whatever).

Reply:
Alright, I see more the target of this document